quantum-adapters/kiwoom-adapter/src/kiwoom_api/functions/new_stock_rights.py

```
@@ -0,0 +1,199 @@
+        print("=== ka10011 신주인수권전체시세요청 테스트 ===")
+        print(f"응답코드: {result.get('Code')}")
+        print(f"헤더: {result.get('Header')}")
+        
+        if result.get('Code') == 200:
+            body = result.get('Body', {})
+            rights_list = body.get('newstk_recvrht_mrpr', [])
+            print(f"신주인수권 종목 수: {len(rights_list)}")
+            
+            if rights_list:
+                first_item = rights_list[0]
+                print(f"첫 번째 종목:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)